Sources/GRPC/GRPCPayload.swift

```diff
@@ -14,9 +14,15 @@
  * limitations under the License.
  */
 import NIO
-import SwiftProtobuf
 
-public protocol Payload {
-    init(serializedByteBuffer: inout NIO.ByteBuffer) throws
-    func serialize(into buffer: inout NIO.ByteBuffer) throws
+public protocol GRPCPayload {
+    init()
+}
+
+extension GRPCPayload {
+    init(serializedByteBuffer: inout NIO.ByteBuffer) throws {
+        self.init()
+    }
+    
+    func serialize(into buffer: inout NIO.ByteBuffer) throws {}
 }
```

Comment:
I updated the code as recommended however, now we are facing an issue with the current way it's implemented. the `GRPCProtoBufPayload` protocol functions aren't being called, and only the ones in the GRPCPayload are being called instead. Which means we are not serializing nor deserializing the buffer